### PR TITLE
Add prettier and fix errors

### PR DESCRIPTION
**Pull request by reubano**
- Add prettier and prettify code
- Fix null object errors

### CODE REVIEW

**Comment by StephanHoyer** [comment]
would you mind looking at the failing tests?

**Comment by reubano** [comment]
Think I've fixed them. I actually have similar preferences as yours, but didn't know about `standard` and that it's basically a `prettier` alternative. I won't be offended if you switch back to it, or update the config.

File: index.js [hunk 1]
```diff
@@ -244,6 +248,10 @@ async function _render(view, options, hooks) {
       vnode.state = omit(component, COMPONENT_PROPS);
       vnode.attrs = component.attrs || view.attrs || {};
 
+      if (!component.view) {
```
Comment (StephanHoyer) [inline index.js]: don't get this check, can you explain?

File: index.js [hunk 2]
```diff
@@ -1,283 +1,291 @@
-'use strict'
```
Comment (StephanHoyer) [inline index.js]: Would be great to make the lint change in another PR.

**Comment by StephanHoyer** [comment]
Sorry for the delay.

Made some inline comments.

File: index.js (hunk 1 above)
Comment (reubano) [reply index.js]: Not sure how or why, but in my case there were times when `component.view` was empty, so it broke the rendering. This is what fixed it for me.

**Comment by StephanHoyer** [comment]
fixed with f18c225
thanks for your work, tho
